Update version to 1.0.3

haensch-mauricio opened the pull request.

Commit:
Update version to 1.0.3

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found some lint.

Here's what I've got...


For **recipe**:

 * When defining a source/url please add a sha256, sha1 or md5 checksum (sha256 preferably).

nicoddemus committed:
Add 256 hash and modernize recipe

* Update noarch definition
* Use host instead of build because of conda-build 3
* Use pip to install package

nicoddemus committed:
Use requirements/build instead of requirements/host

My understanding was incorrect about when to use requirements/host, reverting this change.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

nicoddemus committed:
Revert to using git checkout instead of tarball

`setuptools_scm` requires a full git repository to operate.
